app/build.gradle

```diff
@@ -12,7 +12,7 @@ android {
 
     defaultConfig {
         applicationId "org.openlake.sampoorna"
-        minSdk 21
+        minSdk 26
```

Comment:
We don't want to do this in any case. Look for alternatives if this is a necessity in the current implementations.